src/lib/geojson.js

```diff
@@ -84,3 +87,154 @@ export function immutablyReplaceCoordinate(
     ...coordinates.slice(indexes[0] + 1)
   ];
 }
+
+export function flattenPositions(geometry) {
+  let positions = [];
+  if (geometry.type === 'Point') {
```

Comment:
Nit: I'd personally prefer a `switch (geometry.type)` over a gauntlet of if statements.